app/Rules/RecaptchaV3.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+class RecaptchaV3 implements Rule
```

Comment:
I do not see you adding this validation rule anywhere. I think you need to add this rule to the registration and login controller. Please add this rule to them.

Note:
Make sure you use the `!empty(config('app.recaptcha.site_key'))` check. So if our users not filling the site_key, they won't get errors.